<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeScreen</class>
 <widget class="QWidget" name="WelcomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Willkommen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/logo/logo</normaloff>:/logo/logo</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">.main {
	background: white;
padding: 0px;
margin: 0px;
}
QLabel {
	color: black;
}
QFrame {
	background: white;
	color: black;
}
.lbl_title {
	font-size: 30px;
	background: #F4F4F4;
	color: black;
}
.lbl_title2{
	font-weight: bold;
	background: #F4F4F4;
	color: black;
}
.lbl_logo {
 background: #F4F4F4;
}
.frame_bottom{
	background: white;
}
.frame_top {
	background: white;
}
.frame_left {
	background: #F4F4F4;
}
.frame_db {
	background: white;
	border: none;
}

.pushButton_addDb {
	border: 1px solid rgb(170,0,0);
	border-radius: 8px;
	padding: 10px;
	color: rgb(170,0,0);
	font-weight: bold;
	font-size: 16px;
}
.pushButton_addDb::hover {
	background: rgb(170,0,0);
	color: white;
}
.pushButton_demo {
	border: 1px solid rgb(0,0,200);
	border-radius: 8px;
	padding: 10px;
	color: rgb(0,0,200);
	font-weight: bold;
	font-size: 16px;
}
.pushButton_demo::hover {
	background: rgb(0,0,200);
	color: white;
}</string>
  </property>
  <property name="class" stdset="0">
   <string>main</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_left">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="class" stdset="0">
      <string>frame_left</string>
     </property>
     <widget class="QLabel" name="lbl_title">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>180</width>
        <height>111</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ink Free</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Willkommen 
bei
PennyWise</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="class" stdset="0">
       <string>lbl_title</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_logo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>180</width>
        <height>180</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>180</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>180</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resources/resources.qrc">:/logo/logo</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="class" stdset="0">
       <string>lbl_logo</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_title2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>180</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>...deinem persÃ¶nlichen
Finanzmanager ðŸ’²ðŸ’²ðŸ’²</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="class" stdset="0">
       <string>lbl_title2</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_right">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_top">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>7</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>24</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_start">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Erstelle deine erste PennyWise Datenbank...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_db">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <property name="class" stdset="0">
            <string>frame_db</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_addDb">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string> Datenbank hinzufÃ¼gen</string>
              </property>
              <property name="iconSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="class" stdset="0">
               <string>pushButton_action pushButton_addDb</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_bottom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>198</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>198</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <property name="class" stdset="0">
         <string>frame_bottom</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>7</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>24</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_start_2">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Demo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>... oder starte die Demo mit BeispieldatensÃ¤tzen, 
um einen ersten Ãœberblick erhalten zu kÃ¶nnen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_start_demo">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Demo starten</string>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="class" stdset="0">
            <string>pushButton_action pushButton_demo</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
